src/orbitalMechanics.cpp

```diff
@@ -0,0 +1,405 @@
+#include "mantella_bits/orbitalMechanics.hpp"
+
+// C++ standard library
+#include <cmath>
+#include <iostream>
+#include <stddef.h>
+#include <string>
+
+// Mantella
+#include "mantella_bits/assert.hpp"
+
+namespace mant {
+  namespace itd {
+
+    double degreeToRadian(
+        const double degree) {
+      return degree * (arma::datum::pi / 180.0);
+    }
+
+    std::vector<std::pair<arma::Col<double>::fixed<3>, arma::Col<double>::fixed<3>>> lambert(
+        const arma::Col<double>::fixed<3>& departurePosition,
+        const arma::Col<double>::fixed<3>& arrivalPosition,
+        const double transferTime,
+        const bool clockwiseDirection) {
+      verify(transferTime > 0.0, "lambert: transferTime must be greater than zero.");
+
```

Comment:
What happens if `arma::all(departurePosition - arrivalPosition < 1e-12)` is true?